eventbroadcaster/notifiersink.go

```diff
@@ -29,34 +27,48 @@ type NotifierSink struct {
 	logger   *zap.SugaredLogger
 }
 
-// RegisterNotifierSinks builds notifiers sinks and registers them on the given broadcaster
-func RegisterNotifierSinks(mgr ctrl.Manager, broadcaster record.EventBroadcaster, notifiersConfig eventnotifier.NotifiersConfig, logger *zap.SugaredLogger) (err error) {
-	client := mgr.GetClient()
-
-	notifiers, err := eventnotifier.GetNotifiers(notifiersConfig, logger)
-
+// RegisterNotifierSinks registers notifiers sinks on the given broadcaster
+func RegisterNotifierSinks(mgr ctrl.Manager, broadcaster record.EventBroadcaster, notifiers []eventnotifier.Notifier, logger *zap.SugaredLogger) {
 	for _, notifier := range notifiers {
 		logger.Infof("notifier %s enabled", notifier.GetNotifierName())
 
-		broadcaster.StartRecordingToSink(&NotifierSink{client: client, notifier: notifier, logger: logger})
+		broadcaster.StartRecordingToSink(&NotifierSink{client: mgr.GetClient(), notifier: notifier, logger: logger})
 	}
 
 	corev1Client, _ := corev1client.NewForConfig(mgr.GetConfig())
 
 	broadcaster.StartRecordingToSink(&corev1client.EventSinkImpl{Interface: corev1Client.Events("")})
-
-	return
 }
 
 func (s *NotifierSink) Create(event *corev1.Event) (*corev1.Event, error) {
-	dis, err := s.getDisruption(event)
-	if err != nil {
+	s.logger.Infof("CREATE event received: %s", event.Message)
```

Comment:
is it really necessary? I feel like it would generate a lot of logs
If yes:

```suggestion
	s.logger.Debugf("CREATE event received: %s", event.Message)
```